### ISSUE TITLE
ENH: Disable map rotation for josm users

### ISSUE THREAD

**Issue by wireguy** [hotosm/tasking-manager#2975]
In josm, map panning is done with a right mouse click and drag.

When presented a map in tm4, a josm user may try to pan the map with the same gesture.
This causes the map to rotate, and is not very useful.

This seems to be the code to disable it:
```
// disable map rotation using right click + drag
map.dragRotate.disable();
```

This could be done when editor=josm

Even better would be to make right mouse click drag be equivalent to left mouse click drag, and pan the map.

**Comment by RAytoun** [comment]
I agree with wireguy. It is a bit of a nuisance value and time waster when I switch from JOSM to TM to check something and automatically use the right mouse button to drag. Not only does it rotate but it also flips so it usually ends with a two move correction to get it back to normal view.
